### PR TITLE
sc-106: Sc 106/image preview should not be a broken image

### PR DESCRIPTION
**Pull request by akshay-rakheja**
{Short Description of Change/Situation}

### Specific Changes:
-Added a default Image 
- Added logic to render default image when minting if an image isnt selected
### Related PRs

> If your PR is related to or dependent on PRs in other services, link them here

### Updated Dependencies

> Include a list of newly added or updated dependencies

### Testing

> Description of what testing was done

### Future work

> If this PR is known to break here or in another service, or a follow up is needed to complete a ticket, describe that here

Pre-Merge Checklist:

- [ ] I have tested my changes locally and started web app and server
- [ ] The client compiles without warnings
- [ ] If changes were made to contracts, the contracts compile and deploy to ropsten without warnings

### CODE REVIEW

File: packages/server/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,10277 @@
 {
```
Comment (connie-ho) [inline packages/server/package-lock.json:1]: were you on the Node 14 when you installed these?

File: packages/client/src/components/Minters.jsx [hunk 2]
```diff
@@ -81,14 +93,14 @@ const Minter = () => {
...
-    const { success, status } = await mintNFT(file, name, description);
+    const { status } = await mintNFT(file, name, description);
     setStatus(status);
-    if (success) {
-      setName('');
-      setDescription('');
-      setFile('');
-      //setURL("");
-    }
+    // if (success) {
```
Comment (connie-ho) [inline packages/client/src/components/Minters.jsx:98]: Are we anticipating we'll need this part for later?

Comment (akshay-rakheja) [reply packages/client/src/components/Minters.jsx:98]: Yup might need this for stepper

File: packages/server/package-lock.json (hunk 1 above)
Comment (akshay-rakheja) [reply packages/server/package-lock.json:1]: Not sure.

Comment (connie-ho) [reply packages/server/package-lock.json:1]: Can you try deleting the node mods and package lock, and re installing with node 14? The package lock json might mess things up for others )):

Comment (connie-ho) [reply packages/server/package-lock.json:1]: Or exclude the package lock from the commit since no dependencies were changed

**Review by connie-ho — APPROVED** [review]

**Review by connie-ho — APPROVED** [review]

**Pull request merged**